AlembicImporter/Packages/com.unity.formats.alembic/Runtime/Scripts/Importer/AbcAPI.cs

```diff
@@ -453,86 +414,53 @@ public struct aiSample
-        [DllImport(Abci.Lib)] static extern void aiPolyMeshGetSubmeshSummaries(IntPtr sample, IntPtr dst);
-        [DllImport(Abci.Lib)] static extern void aiPolyMeshFillVertexBuffer(IntPtr sample, IntPtr vbs, IntPtr ibs);
-        [DllImport(Abci.Lib)] static extern void aiSampleSync(IntPtr sample);
-        #endregion
+        public void GetSummary(ref aiMeshSampleSummary dst) { NativeMethods.aiPolyMeshGetSampleSummary(self, ref dst); }
+        public void GetSplitSummaries(PinnedList<aiMeshSplitSummary> dst) { NativeMethods.aiPolyMeshGetSplitSummaries(self, dst); }
+        public void GetSubmeshSummaries(PinnedList<aiSubmeshSummary> dst) { NativeMethods.aiPolyMeshGetSubmeshSummaries(self, dst); }
+        internal void FillVertexBuffer(PinnedList<aiPolyMeshData> vbs, PinnedList<aiSubmeshData> ibs) { NativeMethods.aiPolyMeshFillVertexBuffer(self, vbs, ibs); }
+        public void Sync() { NativeMethods.aiSampleSync(self); }
     }
 
-    public struct aiPointsSample
+    internal struct aiPointsSample
     {
         public IntPtr self;
         public static implicit operator bool(aiPointsSample v) { return v.self != IntPtr.Zero; }
         public static implicit operator aiSample(aiPointsSample v) { aiSample tmp; tmp.self = v.self; return tmp; }
 
-        public void GetSummary(ref aiPointsSampleSummary dst) { aiPointsGetSampleSummary(self, ref dst); }
-        public void FillData(PinnedList<aiPointsData> dst) { aiPointsFillData(self, dst); }
-        public void Sync() { aiSampleSync(self); }
-
-        #region internal
-        [DllImport(Abci.Lib)] static extern void aiPointsGetSampleSummary(IntPtr sample, ref aiPointsSampleSummary dst);
-        [DllImport(Abci.Lib)] static extern void aiPointsFillData(IntPtr sample, IntPtr dst);
-        [DllImport(Abci.Lib)] static extern void aiSampleSync(IntPtr sample);
-        #endregion
+        public void GetSummary(ref aiPointsSampleSummary dst) { NativeMethods.aiPointsGetSampleSummary(self, ref dst); }
+        public void FillData(PinnedList<aiPointsData> dst) { NativeMethods.aiPointsFillData(self, dst); }
+        public void Sync() { NativeMethods.aiSampleSync(self); }
     }
 
 
-    public struct aiProperty
+    internal struct aiProperty
     {
         public IntPtr self;
         public static implicit operator bool(aiProperty v) { return v.self != IntPtr.Zero; }
-
-        #region internal
-        [DllImport(Abci.Lib)] static extern IntPtr aiPropertyGetName(IntPtr prop);
-        [DllImport(Abci.Lib)] static extern aiPropertyType aiPropertyGetType(IntPtr prop);
-        [DllImport(Abci.Lib)] static extern void aiPropertyGetData(IntPtr prop, aiPropertyData oData);
-        #endregion
-    }
-
-    public partial class AbcAPI
-    {
-        [DllImport(Abci.Lib)] public static extern aiSampleSelector aiTimeToSampleSelector(double time);
-        [DllImport(Abci.Lib)] public static extern void aiCleanup();
+        public static bool ToBool(aiProperty v) { return v; }
```

Comment:
What does that do?

Reply:
it's an alternative for `public static implicit operator bool(aiProperty v) { return v.self != IntPtr.Zero; }`

There's a warning for giving people an alternative if they can't use the operator directly for some reason.

Reply:
OK, and I guess there it will call the operator right? "return v" becomes "(bool) return v" which calls the operator. Thanks